adapters/bidmachine/bidmachinetest/supplemental/wrong-host.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Nitpick: can you fix the formatting, specifically the indentation, in this file to match your other json test files?